src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx

```diff
@@ -375,6 +381,21 @@ export const CommentContainer: FunctionComponent<Props> = ({
           comment={comment}
           story={story}
           onClose={toggleShowEditDialog}
+          origin={'EDIT'}
+        />
+      </div>
+    );
+  }
+
+  if (showDeleteDialog) {
+    return (
+      <div data-testid={`comment-${comment.id}`}>
+        <EditCommentFormContainer
+          settings={settings}
+          comment={comment}
+          story={story}
+          onClose={toggleShowDeleteDialog}
+          origin={'DELETE'}
```

Comment:
We already have a reject option for admin and moderator users via the in stream moderation, why was there a delete flow added here?

By design, Coral does not delete comments. The only way for deletion to occur is during a user deletion process. This is a GDPR requirement and is vital to our architecture and compliance.